VideoLocker/build.gradle

```diff
@@ -96,6 +96,7 @@ dependencies {
     compile 'com.android.support:cardview-v7:21.0.0'
     compile 'com.android.support:appcompat-v7:22.1.0'
     compile 'com.android.support:multidex:1.0.0'
+    compile 'com.github.bumptech.glide:glide:3.6.1'
```

Comment:
Ideally, Volley should be removed in the same commit that introduces Glide, as they are both linked. If you wanted to keep it separate from the layout changes, then this should have preferably been done in a preceding commit.
> On 10-Sep-2015, at 1:02 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/build.gradle <https://github.com/edx/edx-app-android/pull/354#discussion_r39134080>:
> 
> > @@ -96,6 +96,7 @@ dependencies {
> >      compile 'com.android.support:cardview-v7:21.0.0'
> >      compile 'com.android.support:appcompat-v7:22.1.0'
> >      compile 'com.android.support:multidex:1.0.0'
> > +    compile 'com.github.bumptech.glide:glide:3.6.1'
> yeah, I plan too, but I'll do that in another commit, that wont be squashed with this one. So that in commit-history we know what, when and how it happened 
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/354/files#r39134080>.
>